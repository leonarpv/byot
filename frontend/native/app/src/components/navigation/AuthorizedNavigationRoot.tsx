import * as React from 'react';
import {AuthorizedNavigationTabs} from './AuthorizedNavigationTabs';

interface Props {}

export const AuthorizedNavigationRoot: React.FC<Props> = (props: Props) => {
  return <AuthorizedNavigationTabs />;
};
