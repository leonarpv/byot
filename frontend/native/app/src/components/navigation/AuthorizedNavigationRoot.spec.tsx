import * as React from 'react';
import {render} from '@testing-library/react-native';
import {AuthorizedNavigationRoot} from './AuthorizedNavigationRoot';

describe('<AuthorizedNavigationRoot/>', () => {
  it('should render', () => {
    const {asJSON} = render(<AuthorizedNavigationRoot />);
    expect(asJSON()).toMatchSnapshot();
  });
});
