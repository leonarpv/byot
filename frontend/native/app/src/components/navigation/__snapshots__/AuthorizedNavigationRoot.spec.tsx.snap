// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthorizedNavigationRoot/> should render 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    initialRouteName="TrainingSetList"
  >
    <View
      name="TrainingSetList"
    />
  </View>
</View>
`;
